package.json

```diff
@@ -293,6 +294,7 @@
         "react-tether": "1.0.5",
         "react-textarea-autosize": "^8.5.3",
         "regenerator-runtime": "^0.13.3",
+        "remark-gfm": "^4.0.0",
```

Comment:
Needed to install this to get the tables to render correctly 
https://storybook.js.org/docs/writing-docs/mdx#markdown-tables-arent-rendering-correctly